src/frontend/web_application/src/components/MessageList/components/Message/index.jsx

```diff
@@ -1,21 +1,22 @@
 import React, { PropTypes, Component } from 'react';
+import { v1 as uuidV1 } from 'uuid';
 import classnames from 'classnames';
 import { DateTime } from '@gandi/react-translate';
 import ContactAvatarLetter from '../../../ContactAvatarLetter';
+import Dropdown, { withDropdownControl } from '../../../../components/Dropdown';
 import Button from '../../../Button';
 import Icon from '../../../Icon';
 import MessageActionsContainer from '../MessageActionsContainer';
 
 import './style.scss';
 
+const DropdownControl = withDropdownControl(Button);
 
 const MessageInfosContainer = ({ __, message, author }) => (
   <div className="m-message__infos-container">
-    <div className="m-message__author">
-      {author.address}
-    </div>
+    <div className="m-message__author">{author.address}</div>
     <div className="m-message__type">
-      {__('by')} {message.type} <Icon type={message.type} className="m-message__type-ico" spaced />
+      <span className="m-message__type__label">{__('by')} {message.type}</span> <Icon type={message.type} className="m-message__type__icon" spaced />
```

Comment:
only 2 levels: the block `m-message` and the element (witch can be nested) e.g. `type` and `type-label`

And you can add unbreakable spaces with `{' '}` so you can add line breaks.